modules/rest/src/testFixtures/java/org/apache/ignite/internal/rest/matcher/ProblemMatcher.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.rest.matcher;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import java.util.Collection;
+import java.util.UUID;
+import org.apache.ignite.internal.rest.api.InvalidParam;
+import org.apache.ignite.internal.rest.api.Problem;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Matcher for {@link Problem}.
+ */
+public class ProblemMatcher extends TypeSafeMatcher<Problem> {
+    private Matcher<String> titleMatcher;
+
+    private Matcher<Integer> statusMatcher;
+
+    private Matcher<String> codeMatcher;
+
+    private Matcher<String> typeMatcher;
+
+    private Matcher<String> detailMatcher;
+
+    private Matcher<String> nodeMatcher;
+
+    private Matcher<UUID> traceIdMatcher;
+
+    private Matcher<Collection<InvalidParam>> invalidParamsMatcher;
+
+    /**
+     * Creates a matcher for {@link Problem}.
+     *
+     * @return Matcher.
+     */
+    public static ProblemMatcher isProblem() {
```

Comment:
JobStatus matcher uses builder-like style but here you prefer another one. Can you use the same style for all matchers?